extensions/2.0/Khronos/KHR_spz_gaussian_splats_compression/README.md

```diff
@@ -0,0 +1,416 @@
+
+### Schema Example
+
+Example SPZ extension shown below. This extension only affects any `primitive` nodes containting Gaussian splat data.
+
+```json
+  "meshes": [{
+      "primitives": [{
+          "attributes": {
+            "POSITION": 0,
+            "COLOR_0": 1,
+            "_SCALE": 2,
+            "_ROTATION": 3,
+            "_SH_DEGREE_1_COEF_0": 4,
+            "_SH_DEGREE_1_COEF_1": 5,
+            "_SH_DEGREE_1_COEF_2": 6
+          },
+          "material": 0,
+          "mode": 0,
+          "extensions": {
+            "KHR_spz_gaussian_splats_compression": {
+              "bufferView": 0,
+            }
+          }
+        }]
+    }],
+  "buffers": [{
+      "uri": "0.bin",
+      "byteLength": 9753142
+    }],
+  "bufferViews": [{
+      "buffer": 0,
+      "byteLength": 9753142
+    }],
+```
+
+### Extension Properties
+
+#### bufferView
+
+This property points to the bufferView containing the Gaussian splat data compressed with SPZ.
+
+#### attributes
+
+This contains the attributes that will map into the compressed SPZ data indicated by `bufferView`. At minimum it will contain `POSITION`, `COLOR_0`, `_ROTATION`, and `_SCALE`. `_SH_DEGREE_ℓ_COEF_n` attributes hold the spherical harmonics data and are not required. If higher degrees are used then lower degrees are required implicitly.
+
+| Splat Data | glTF Attribute | Accessor Type | Component Type | Required | Notes |
+| --- | --- | --- | --- | --- | --- |
+| Position | POSITION | VEC3 | float | yes | |
+| Color (Spherical Harmonic degree 0 (Diffuse) and alpha) | COLOR_0 | VEC4 | unsigned byte normalized or float | yes | |
+| Rotation | _ROTATION | VEC4 | float | yes | Rotation is a quaternion. |
+| Scale | _SCALE | VEC3 | float | yes | |
+| Spherical Harmonics degree 1 | _SH_DEGREE_1_COEF_n (n = 0 to 2) | VEC3 | float | no (yes if degree 2 or 3 are used) | |
+| Spherical Harmonics degree 2 | _SH_DEGREE_2_COEF_n (n = 0 to 4) | VEC3 | float | no (yes if degree 3 is used) | |
+| Spherical Harmonics degree 3 | _SH_DEGREE_3_COEF_n (n = 0 to 6) | VEC3 | float | no | |
+
+Each increasing degree of spherical harmonics requires more coeffecients. At the 1st degree, 3 sets of coeffcients are required, increasing to 5 sets for the 2nd degree, and increasing to 7 sets at the 3rd degree. With all 3 degrees, this results in 45 spherical harmonic coefficients stored in the `_SH_DEGREE_ℓ_COEF_n` attributes.
+
+**TODO:** SPZ to attributes diagram
+
+### Accessors
+
+Required `accessors` for `POSITION`, `COLOR_0`, `_ROTATION`, and `_SCALE`:
+
+```json
+  "accessors": [{
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC3",
+      "max": [
+        1,
+        1,
+        1,
+      ],
+      "min": [
+        -1,
+        -1,
+        -1,
+      ]
+    }, {
+      "componentType": 5121,
+      "count": 590392,
+      "type": "VEC4",
+      "normalized": true
+    }, {
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC3"

```

Comment:
If this example is showing rotation then scale, it should be `VEC4` for rotation.

```suggestion
      "type": "VEC4"

```

Reply:
Actually, these two accessors are swapped in the statement on line 124. I want to make sure that the order corresponds to the example given on line 67.